app/views/eventRegistrations.html

```diff
@@ -44,17 +44,34 @@ <h2 class="page-title" translate>Registrations</h2>
           </button>
           <button class="btn btn-primary" ng-click="refreshRegistrations()"><i class="fa fa-refresh"></i> <translate>Refresh</translate></button>
           <button type="button" class="btn btn-primary dropdown-toggle" dropdown-toggle>
-            <translate>Columns</translate>
+            <span class="hidden-xs"><translate>Display</translate></span><translate> Columns</translate>
```

Comment:
Looks like you need to put a space outside the `<translate>` tag. angular-gettext which does the translating seems to strip whitespace.
I'd put it here:
```
<span class="hidden-xs"><translate>Display</translate></span> <translate>Columns</translate>
                                                             ^
```